test/function.c

```diff
@@ -4,14 +4,14 @@
 int foo(int bar, int baz);
 
 /**
- * Void parameter.
+ * No parameters.
  */
-int fooing(void);
+int no_parameters(void);
 
 /**
- * Empty parameters.
+ * Empty parameter list.
```

Comment:
'unknown' might be more descriptive, no?

Reply:
The standard talks about "empty list".